platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/ConsensusEvent.java

```diff
@@ -37,5 +38,15 @@ public interface ConsensusEvent extends Event, ReachedConsensus {
      * @return a consensus transaction iterator
      */
     @NonNull
+    @Deprecated
```

Comment:
add a `@deprecated` javadoc comment explaining apps should switch to the new method

Reply:
Following your comment below I think we can just keep the existing method. The ConsensusTransaction needs to be just updated to return Bytes for a given transaction.